ui/packages/consul-ui/app/components/code-editor/layout.scss

```diff
@@ -32,3 +33,121 @@
 %code-editor > pre {
   display: none;
 }
+
+%code-editor .ember-basic-dropdown-trigger {
+  background-color: var(--tone-gray-900);
+  color: var(--black);
+  border-radius: 2px;
+  border: 1px solid;
```

Comment:
Not super important but we have some more props for these if you could use those

https://github.com/hashicorp/consul/blob/main/ui/packages/consul-ui/app/styles/base/decoration/base-variables.scss#L4-L10

Documentation for all this coming soon!